packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -61,18 +64,18 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   const errorBarProps = usePropsFor(ErrorBar);
 
   const onRenderAvatarCallback = useCallback(
-    (userId, options) => {
-      return <AvatarPersona userId={userId} {...options} dataProvider={onFetchAvatarPersonaData} />;
+    (userId, defaultOptions) => {
+      return <AvatarPersona userId={userId} {...defaultOptions} dataProvider={onFetchAvatarPersonaData} />;
     },
     [onFetchAvatarPersonaData]
   );
 
   return (
     <Stack className={chatContainer} grow>
-      {hiddenElements?.topic !== true && <ChatHeader {...headerProps} />}
+      {options?.topic !== false && <ChatHeader {...headerProps} />}
```

Comment:
these are now double negative checks, can they be simplified to 
```suggestion
      {options?.topic && <ChatHeader {...headerProps} />}
```

Reply:
This won't work if topic is undefined or null

Reply:
Ah yeah you're right because if its undefined or null we want to show the ChatHeader - thanks !